.eslintrc

```diff
@@ -2,6 +2,13 @@
   "extends": ["airbnb", "plugin:prettier/recommended"],
   "plugins": ["dependencies"],
   "parser": "babel-eslint",
+  "settings": {
+    "import/resolver": {
```

Comment:
Webpack has been configured so that we don't have to enter long relative paths eg ../../../ After this change, eslint displayed errors with these imports because they considered them to be wrong. This record tells eslint which file contains the appropriate settings.

Reply:
Webpack has been configured so that you do not have to enter long relative paths like ../../../ After this change, eslint displayed errors with these imports because they considered them to be wrong. This record tells eslint which file contains the appropriate settings.